### PR TITLE
Added strimzi network policy

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

File: controllers/cloud.redhat.com/providers/kafka/strimzi.go [hunk 1]
```diff
@@ -496,9 +500,62 @@ func NewStrimzi(p *providers.Provider) (providers.ClowderProvider, error) {
 		},
 	}
 
+	createNetworkPolicies(p)
```
Comment (kylape) [inline controllers/cloud.redhat.com/providers/kafka/strimzi.go:503]: you don't check the error

File: controllers/cloud.redhat.com/providers/kafka/strimzi.go [hunk 2]
```diff
@@ -496,9 +500,62 @@ func NewStrimzi(p *providers.Provider) (providers.ClowderProvider, error) {
...
+	appList, err := p.Env.GetAppsInEnv(p.Ctx, p.Client)
+	if err != nil {
+		return err
+	}
+
+	namespaceSet := map[string]bool{}
+
+	np := &networking.NetworkPolicy{}
+	nn := types.NamespacedName{
+		Name:      p.Env.Name,
```
Comment (kylape) [inline controllers/cloud.redhat.com/providers/kafka/strimzi.go:518]: should we make it the same name as the kafka instance?  i.e. `getKafkaName(p.Env)`?

File: controllers/cloud.redhat.com/providers/kafka/strimzi.go [hunk 3]
```diff
@@ -496,9 +500,62 @@ func NewStrimzi(p *providers.Provider) (providers.ClowderProvider, error) {
...
+
+	for _, app := range appList.Items {
+		if _, ok := namespaceSet[app.Namespace]; ok {
+			continue
+		}
+
+		npFrom = append(npFrom, networking.NetworkPolicyPeer{
+			NamespaceSelector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"kubernetes.io/metadata.name": app.Namespace,
```
Comment (kylape) [inline controllers/cloud.redhat.com/providers/kafka/strimzi.go:536]: does this work with the version of k8s we're running in stage/prod?

**Review by kylape — CHANGES_REQUESTED** [review]

File: controllers/cloud.redhat.com/providers/kafka/strimzi.go (hunk 3 above)
Comment (psav) [reply controllers/cloud.redhat.com/providers/kafka/strimzi.go:536]: What do you mean does this work with?

File: controllers/cloud.redhat.com/providers/kafka/strimzi.go (hunk 1 above)
Comment (psav) [reply controllers/cloud.redhat.com/providers/kafka/strimzi.go:503]: yikes - it was a todo - and I forgot

**Review by kylape — APPROVED** [review]

**Pull request merged**